code/modules/sanity/sanity_mob.dm

```diff
@@ -59,12 +59,14 @@
 	var/breakdown_time = 0
 	var/spook_time = 0
 
-	var/list/datum/breakdown/breakdowns = list()
+	var/list/datum/breakdown/breakdowns
 
 /datum/sanity/New(mob/living/carbon/human/H)
 	owner = H
 	level = max_level
 	insight = rand(0, 30)
+	breakdowns = list()
+	desires = list()
```

Comment:
Why do this? Does list init of two lists in (init) cause noticeable performance issues?

Reply:
No but it's a good habit.

From the /tg/ contributing.MD

>Defining a list in the variable's definition calls a hidden proc - init. If you have to define a list at startup, do so in New() (or preferably Initialize()) and avoid the overhead of a second call (Init() and then New())

Reply:
I'd say that the benefit is completely cancelled out by the code being harder to read.